src/language-js/print/type-parameters.js

```diff
@@ -32,12 +33,25 @@ function printTypeParameters(path, options, print, paramsKey) {
   const grandparent = path.getNode(2);
   const isParameterInTestCall = grandparent && isTestCall(grandparent);
 
+  const isArrowFunctionVariable = path.match(
```

Comment:
I don't like that there are two different `isArrowFunctionVariable` checks in two places. Looks like something that can get out of sync one day. Any idea how to improve this?